src/components/CopyInUserDialog.component.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import PropTypes from "prop-types";
+import BatchModelsMultiSelectModel from "./batch-models-multi-select/BatchModelsMultiSelect.model";
+import BatchModelsMultiSelectComponent from "./batch-models-multi-select/BatchModelsMultiSelect.component";
+import _m from "../utils/lodash-mixins";
+
+function getTitle(getTranslation, users) {
+    const usernames = users && users.map(user => user.userCredentials.username);
+    const info = usernames ? _m.joinString(getTranslation, usernames, 3, ", ") : "...";
+    return getTranslation("copyInUser") + ": " + info;
+}
+
+function getPayload(pairs) {
```

Comment:
Logic functions should be in helpers (usersHelper or a new one you create). The idea is to keep code in the components relevant only to React.